src/components/core/Input/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classes from './Input.module.scss';
+
+interface InputProps {
+  type: string;
+  placeholder: string;
+  id: string;
+  name: string;
+  label: string;
```

Comment:
Here I would suggest setting label to be optional param, because sometimes we want only input field without it